setup/setup_fedora.sh

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env bash
+
+echo "Beginning with Fedora setup. Have a walk while the machine is setup"
+
+# Upgrade the already present packages
+sudo dnf -y upgrade
+
+# Install zsh
+sudo dnf -y install zsh
+
+# Install everything else
+sudo dnf -y install ack
+sudo dnf -y install git git-extras hub 
+sudo dnf -y install htop ngrep nmap
+sudo dnf -y install autojump # A smart command to change directory (https://github.com/wting/autojump)
+pip install legit            # http://www.git-legit.org/
+legit install		     # Enables the extra git commands
+sudo dnf -y install zopfli   # Data compression software (https://github.com/google/zopfli)
+sudo dnf -y install cowsay
+# pup not available for Linux (Fedora)
+sudo dnf -y install sshrc    # https://github.com/Russell91/sshrc
+pip install stormssh         # Linux alternative for storm https://github.com/emre/storm
+pip install --user howdoi
+sudo dnf -y install httpie   # https://github.com/jakubroztocil/httpie
+sudo dnf -y install jq       # https://stedolan.github.io/jq/
+sudo dnf -y install elixir
+sudo dnf -y install ShellCheck
+sudo dnf -y install teamviewer
+
+###############################################################################
+#                        Install utilities                                    #
+###############################################################################
+
+sudo dnf -y install firefox
+#Evernote,skitch,dash not available
+sudo dnf -y install https://prerelease.keybase.io/keybase_amd64.rpm
+#android-file-transfer-linux http://whoozle.github.io/android-file-transfer-linux/
```

Comment:
Please use a space after `#`, that makes it more readable and for text editors to easily comment/un-comment lines.